crates/quickjs-wasm-rs/README.md

```diff
@@ -27,3 +27,23 @@ let input_value = messagepack::transcode_input(&context, input_bytes).unwrap();
 let output_value: Value = ...;
 let output = messagepack::transcode_output(output_value).unwrap();
 ```
+
+## Building a project using this crate
+
+- Install the [wasi-sdk](https://github.com/WebAssembly/wasi-sdk#install) for your platform
+- Set the `QUICKJS_WASM_SYS_WASI_SDK_PATH` environment variable to the absolute path where you installed the `wasi-sdk`
+
+For example, if you install the `wasi-sdk` in `/opt/wasi-sdk`, you can run:
+```bash
+export QUICKJS_WASM_SYS_WASI_SDK_PATH=/opt/wasi-sdk
+```
+
+## Publishing to crates.io
+
+To publish this crate to crates.io, you will need to ensure the `QUICKJS_WASM_SYS_WASI_SDK_PATH` is set to a value pointing to the absolute path where you installed the `wasi-sdk`. The `--target` parameter will also need to be set to `wasm32-wasi`.
+
+E.g.,
+
+```
+QUICKJS_WASM_SYS_WASI_SDK_PATH=/opt/wasi-sdk cargo publish --target=wasm32-wasi
```

Comment:
I can do that 👍. I'll still need to document that we need to run the script because a barebones `cargo publish` fails with a misleading error message.